src/tests/support.cpp

```diff
@@ -103,6 +103,19 @@ file_contents(const char *path, ssize_t *size)
     return mem;
 }
 
+int
```

Comment:
I know you're just moving this code but I think this should be `off_t`. I know on my system `st_size` is 8 bytes and int is 4.

Reply:
@dewyatt  Agree on this. Just updated the PR.